test/atom-test.ts

```diff
@@ -1,5 +1,5 @@
 import FeedMe from '..';
-import fs     from 'fs';
+import fs     from 'node:fs';
 import path   from 'path';
 import assert from 'assert';
```

Comment:
why not change `path` and `assert` modules too? they may not be to be polyfilled, but we should be consistent with the way core modules are imported